portality/models/openurl.py

```diff
@@ -29,7 +29,9 @@
     'isbn' : ('index.issn.exact', 'index.issn.exact'),
 }
 
-TERMS_SEARCH = { "query" : {"bool" : { "must" : [] } } }
+# Terms search template. Ensure all queries from OpenURL return publicly visible results with in_doaj : true
+IN_DOAJ_TERM = { "term" : { "admin.in_doaj" : True } }
+TERMS_SEARCH = { "query" : {"bool" : { "must" : [ IN_DOAJ_TERM ] } } }
```

Comment:
@richard-jones I've noticed you do filtered queries, should we do this or is this OK?